src/json-validator.cpp

```diff
@@ -1276,6 +1281,17 @@ std::shared_ptr<schema> schema::make(json &schema,
 			// so this is the origial URI for this reference, the $ref-value has thus be resolved from it
 			auto id = uris.back().derive(attr.value().get<std::string>());
 			sch = root->get_or_create_ref(id);
+
+			const auto defaultAttr = schema.find("default");
```

Comment:
lower-case snake-case please
```suggestion
			const auto default_attr = schema.find("default");
```